tests/BinaryOperators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -330,8 +326,7 @@ contract ArithmeticOperators {
                 veryVeryVeryVeryVeryLongVariableCalledB;
         veryVeryVeryVeryVeryLongFunctionCalledA(
             veryVeryVeryVeryVeryLongVariableCalledB
-        ) **
-            c;
+        )**c;
```

Comment:
just noticing that there is no space before `**` and after it; is that what we want?

Reply:
this was applied in #174

Reply:
can we approve this PR?